trunk/SUAVE/Analyses/Weights/Weights_eVTOL.py

```diff
@@ -0,0 +1,126 @@
+## @ingroup Analyses-Weights
+
+# Weights_eVTOL.py
+#
+# Created:  Aug, 2017, J. Smart
+# Modified: Apr, 2018, J. Smart
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+import SUAVE
+from SUAVE.Core import Data
+from .Weights import Weights
+
+
+# ----------------------------------------------------------------------
+#  Analysis
+# ----------------------------------------------------------------------
+
+## @ingroup Analyses-Weights
+class Weights_eVTOL(Weights):
+    """ SUAVE.Analyses.Weights.Weights_eVTOL()
+    
+    Assumptions:
+    None
+    
+    Source:
+    N/A
+    
+    Inputs:
+    N/A
+    
+    Outputs:
+    N/A
+    
+    Properties Used:
+    N/A
+    """
+
+    def __defaults__(self):
+        """Sets the default parameters for the weight analysis
+
+        Assumptions:
+        None
+
+        Source:
+        N/A
+
+        Inputs:
+        N/A
+
+        Outputs:
+        N/A
+
+        Properties Used:
+        N/A
+        """
+        
+        self.tag = 'weights_eVTOL'
+        
+        self.vehicle  = Data()
+        self.settings = Data()
+        
+        
+    def evaluate(self,conditions=None):
```

Comment:
Go take a look at the other weights analyses. You only should need to set the defaults for the weights function. The rest of this file is a waste of space.

Reply:
Thanks, updated the file